eoscostarica/eosio-spend-explorer
# Feat/delegate claimed and unclaimed graphs

### Delegate incomes claimed and unclaimed graphs

### What does this PR do?

- Resolve #99 
- Resolve #74

### Steps to test

1. Run project
2. Go to Income
3. You should see this:
![imagen](https://user-images.githubusercontent.com/49013556/190532220-4256c4c0-679d-464b-aee0-9503d0831322.png)


#### CheckList
